docs/source/ddp_comm_hooks.rst

```diff
@@ -9,13 +9,6 @@ and users can easily apply any of these hooks to optimize communication.
 Besides, the hook interface can also support user-defined communication
 strategies for more advanced use cases.
 
-.. warning ::
-    DDP communication hook is experimental and subject to change.
```

Comment:
do we want to make DDP stable for 1.9?

Reply:
I am discussing this with @pritamdamania87  There is no more blocker as I see.